test/func/fixtures/jquery.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fixture</title>
+  </head>
+  <body>
+    <div class="e2e-content"></div>
+    <div class="e2e-error"></div>
+    <script src="error-handler.js"></script>
+    <script src="../../../lib/little-loader.js"></script>
+    <script src="jquery/jquery.js"></script>
```

Comment:
`jquery/jquery.js` here is the "execute the jquery loads script", not jquery itself ;)